user/zig/template.py

```diff
@@ -26,14 +26,12 @@
 license = "MIT"
 url = "https://github.com/ziglang/zig"
 source = f"https://ziglang.org/download/{pkgver}/zig-{pkgver}.tar.xz"
-sha256 = "06c73596beeccb71cc073805bdb9c0e05764128f16478fa53bf17dfabc1d4318"
+sha256 = "d9b30c7aa983fcff5eed2084d54ae83eaafe7ff3a84d8fb754d854165a6e521c"
 # lighten up the build, only applies to bootstrap and just slows down the build
 tool_flags = {"CFLAGS": ["-U_FORTIFY_SOURCE"]}
 hardening = ["!int", "!scp", "!ssp", "!var-init"]
 options = ["!lto"]
 
-restricted = "work in progress (needs to either not need llvm or for us to multiversion llvm)"
```

Comment:
this should remain restricted until the llvm thing is properly figured out (aligning with the current version of llvm is not enough to unrestrict it, even temporarily)